WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/InPersonPaymentsViewModel.swift

```diff
@@ -63,6 +63,12 @@ final class InPersonPaymentsViewModel: ObservableObject, PaymentSettingsFlowPres
         useCase.installCardPresentPlugin()
     }
 
+    ///
+    ///
```

Comment:
Oops, I'll remove this before merging, so it doesn't have to run CI again 😅

Reply:
Addressed here: https://github.com/woocommerce/woocommerce-ios/pull/10616/commits/2967e2919c51b7d2c33bcf50f512e50219f392cb